src/explorer/utils/validations.ts

```diff
@@ -21,12 +21,12 @@ export function inputValidation(value: string, key: string): string {
     "total_cru",
     "pricingPolicyId",
   ];
-  const textualFields: string[] = ["farmingPolicyName", "certificationType", "farmName", "farm_name"];
+  const textualFields: string[] = ["farmingPolicyName", "certificationType", "farmName"];
+  const textNumbersFields: string[] = ["farm_name"];
```

Comment:
what is the difference between the `farm_name` and `farmName`? and why they are treated differently